wiki/Submission/Beatmap_Ranking_Procedure/en.md

```diff
@@ -1,41 +1,44 @@
-<img src="Ranking procedure.png" title="Ranking flowchart." alt="Ranking flowchart." width="270" />
+Beatmap Ranking Procedure
+============================
+
+![Ranking flowchart.](Ranking_procedure.png "Ranking flowchart.")
 
 When a beatmap is first made, it is in the Pending or Work in Progress sections of the forum. These maps are referred to as "unranked". Unranked maps do not have a scoreboard and go to the Graveyard section of the forum after a period of inactivity. In order for a map to have a scoreboard and to be moved to the "Ranked" section of the forums, a map must go through the beatmap approval process. First it must get bubbled, and then ranked. Only BNs can bubble and subsequently rank a beatmap.
 
-Bubble ![](thinking.gif "fig:thinking.gif")
--------------------------------------------
+Bubble ![Thinking icon](/wiki/shared/Thinking.gif "Thinking icon")
+--------------------------------------------------------------------
 
-Before a beatmap can become ranked by a [BN](Beatmap_Nomination_Group "wikilink"), it must first get bubbled. In order for a [beatmap](beatmap "wikilink") to become bubbled, it must follow and be in accordance with the [ranking criteria](ranking_criteria "wikilink"), and it must have at least 12 [Star Priority](Star_Priority "wikilink") (or SP). You can actively gain SP in two different ways. First, you can have others help mod your map. If you find their mod to be useful and give kudosu, you will gain 1 SP. You and/or others can also spend [kudosu](kudosu "wikilink") in order to "star" your map (A star costs 1 kudosu, and can be done through either the [mapper](mappers "wikilink")'s profile or through the [map](beatmap "wikilink")'s thread). Once a beatmap has at least 12 SP, a [BN](Beatmap_Nomination_Group "wikilink") can bubble it if they feel it meets two conditions:
+Before a beatmap can become ranked by a [BN](/wiki/People/Beatmap_Nomination_Group), it must first get bubbled. In order for a [beatmap](/wiki/Beatmaps) to become bubbled, it must follow and be in accordance with the [ranking criteria](/wiki/Ranking_Criteria), and it must have at least 12 [Star Priority](/wiki/Glossary/#star-priority) (or SP). You can actively gain SP in two different ways. First, you can have others help mod your map. If you find their mod to be useful and give kudosu, you will gain 1 SP. You and/or others can also spend [kudosu](/wiki/Glossary/#kudosu!) in order to "star" your map (A star costs 1 kudosu, and can be done through either the [mapper](/wiki/Glossary/#beatmapper)'s profile or through the [map](/wiki/Beatmaps)'s thread). Once a beatmap has at least 12 SP, a [BN](/wiki/People/Beatmap_Nomination_Group) can bubble it if they feel it meets two conditions:
```

Comment:
remove the second BN link (there already is one near the start of this paragraph)